### PR TITLE
Added Basket cells and subclasses

### LINKED ISSUE #368: NTR: basket cell morphology
**Term:** 
basket cell morphology

**Definition:**
A cell morphology that inheres in neurons that have sparsely spinous dendrite, which spread radially and tangentially, with axons on the pial side of the soma ascending or descending to give off horizontal collaterals, which in turn sprout small branches ending in dense tufts of terminals.
doi:10.1016/b978-0-12-369497-3.10004-4

**Parent Class:**
Cell Morphology #364 

**Other Notes:**
Neocortex Basket Cell: nifext:56; ILX:0107351.  Basket Cell Morphology: nifext:3; ILX:0101123 
ilxtr:BasketPhenotype

### LINKED ISSUE #369: NTR: small basket cell morphology
**Term:** 
small basket cell morphology

**Definition:**
A basket cell morphology that inheres in neurons that are large, aspiny and multipolar, and gives rise to many long horizontally and vertically projecting axon collaterals that traverse neighbouring columns
PMID:11884355

**Parent Class:**
basket cell morphology #368 

**Other Notes:**
ilxtr:SmallBasketPhenotype

### LINKED ISSUE #370: NTR: large basket cell morphology
**Term:** 
large basket cell morphology

**Definition:**
A basket cell morphology that inheres in neurons that have axonal arbors, composed of frequent short, curvy axonal branches that tend to be near their somata and within the same cortical layer.
PMID:11884355

**Parent Class:**
basket cell morphology #368 

**Other Notes:**
ilxtr:LargeBasketPhenotype

### LINKED ISSUE #371: NTR: nest basket cell morphology
**Term:** 
nest basket cell morphology

**Definition:**
A basket cell morphology that inheres in neurons which have simpler dendritic arbors (compared to small or large basket cell morphology), and an axonal plexus of intermediate density, composed of a few long, smooth axonal branches.
PMID:11884355

**Parent Class:**
basket cell morphology #368 

**Other Notes:**
ilxtr:NestBasketPhenotype

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #368
Fixes #369
Fixes #370
Fixes #371

### CODE REVIEW

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -21847,6 +21847,40 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
 is_a: PATO:0002248 ! pigmented
 created_by: https://orcid.org/0000-0002-3528-5267
 
+[Term]
+id: PATO:0070002
+name: basket cell morphology
+def: "A cell morphology that inheres in neurons that have sparsely spinous dendrite, which spread radially and tangentially, with axons on the pial side of the soma ascending or descending to give off horizontal collaterals, which in turn sprout small branches ending in dense tufts of terminals." [doi:10.1016/b978-0-12-369497-3.10004-4]
```
Comment (patrick-lloyd-ray) [inline src/ontology/pato-edit.obo:21853]: First instance of 'dendrite' should be 'dendrites'.

File: src/ontology/pato-edit.obo [hunk 2]
```diff
@@ -21847,6 +21847,40 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
...
+xref: ILX:0101123
+xref: ilxtr:BasketPhenotype
+xref: nifext:3
+is_a: PATO:0010006 ! cell morphology
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070003
+name: small basket cell morphology
+def: "A basket cell morphology that inheres in neurons that are large, aspiny and multipolar, and gives rise to many long horizontally and vertically projecting axon collaterals that traverse neighbouring columns." [PMID:11884355]
```
Comment (patrick-lloyd-ray) [inline src/ontology/pato-edit.obo:21863]: This one is a bit difficult for me to parse. What is it that "gives rise to many long horizontally and vertically projecting axon collaterals"? The morphology?

**Review by patrick-lloyd-ray — COMMENTED** [review]
A couple of line comments.

File: src/ontology/pato-edit.obo (hunk 2 above)
Comment (shawntanzk) [reply src/ontology/pato-edit.obo]: my bad, I've gone to the paper and copied over the correct def for this

File: src/ontology/pato-edit.obo [hunk 3]
```diff
@@ -21847,6 +21847,40 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
...
+xref: ilxtr:BasketPhenotype
+xref: nifext:3
+is_a: PATO:0010006 ! cell morphology
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070003
+name: small basket cell morphology
+def: "A basket cell morphology that inheres in neurons that are aspiny and multipolar, with axonal arbors, composed of frequent short, curvy axonal branches, that tend to be near their somata and within the same layer." [PMID:11884355]
+xref: ilxtr:SmallBasketPhenotype
```
Comment (dosumis) [inline src/ontology/pato-edit.obo:21864]: 'aspiny' seems to contradict assertion on parent class 'sparsely spinous'.  Maybe 'basket cell' should be defined as aspiny of sparsely spiny.

Punctuation needs fixing:

> with axonal arbors, composed of frequent short, curvy axonal branches, that tend to be near their somata and within the same layer.

-> 
with axonal arbors composed of frequent, short, curvy axonal branches that tend to be near their somata and within the same layer." [PMID:11884355]

**Review by dosumis — CHANGES_REQUESTED** [review]

**Comment by shawntanzk** [comment]
@dosumis sorry, accidentally ask for re-review, have not fixed it yet, will let you know when I do

File: src/ontology/pato-edit.obo (hunk 3 above)
Comment (shawntanzk) [reply src/ontology/pato-edit.obo:21880]: Fixed

**Review by patrick-lloyd-ray — DISMISSED** [review]

**Review by dosumis — CHANGES_REQUESTED** [review]
see comments

**Review by dosumis — APPROVED** [review]

**Pull request merged**